test/box-unbox.js

```diff
@@ -324,12 +338,13 @@ function run () {
     const ciphertext = Buffer.from(JSON.stringify(content)).toString('base64') + '.box.hah'
 
     feed.publish(ciphertext, (_, msg) => {
-      t.true(initDone, 'unboxer completed initialisation before publish')
+      t.true(boxerInitDone, 'boxer completed initialisation before publish')
+      t.false(unboxerInitDone, 'unboxer did not completed initialisation before publish')
```

Comment:
EXTRA ATTENTION HERE: I changed a test. I don't think I made a mistake, but changing tests is always kinda scary. :eyes: 

Previously we made sure that the unboxer init was done after publishing, but now that they're decoupled we don't want to enforce this constraint. Instead, we want to ensure that the **boxer** init is done, and (since I've made the unboxer init take twice as long) we want to ensure that the **unboxer** init is not done.

Reply:
oh dear ... see comment below